packages/sdk/src/types/ViewModeTypes.ts

```diff
@@ -0,0 +1,4 @@
+export enum ViewMode {
```

Comment:
Can u plz export this so it can be used in studio workspace and other integrations